bentoml/_internal/frameworks/picklable_model.py

```diff
@@ -0,0 +1,272 @@
+
+    _model = Model.create(
+        name, module=MODULE_NAME, metadata=metadata, context=context, options=options
+    )
+
+    with open(_model.path_of(f"{SAVE_NAMESPACE}{PKL_EXT}"), "wb") as f:
+        cloudpickle.dump(obj, f)
+
+    _model.save(model_store)
+    return _model.tag
+
+
+class _PicklableModelRunner(Runner):
+    @inject
+    def __init__(
+        self,
+        tag: Tag,
+        function_name: str,
+        name: str,
+        model_info: "Model",
+        model_file: PathType,
+        model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
+    ):
+        super().__init__(name, {}, {})
+        self._model_store = model_store
+        self._model_info = model_info
+        self._model_file = model_file
+        self._function_name = function_name
+
+    @property
+    def num_replica(self) -> int:
+        return max(int(self.resource_quota.cpu), 1)
+
+    @property
+    def required_models(self) -> t.List[Tag]:
+        return [self._model_info.tag]
+
+    # pylint: disable=attribute-defined-outside-init
+    def _setup(self) -> None:
+        with open(self._model_file, "rb") as f:
+            self._model = pickle.load(f)
+        self._infer_func = getattr(self._model, self._function_name)
+
+    # pylint: disable=arguments-differ
+    def _run_batch(  # type: ignore[reportIncompatibleMethodOverride]
+        self,
+        inputs: t.Any,
+    ) -> t.Any:
+        return self._infer_func(inputs)
+
+
+class _PicklableModelSimpleRunner(SimpleRunner):
+    @inject
+    def __init__(
+        self,
+        tag: Tag,
+        function_name: str,
+        name: str,
+        model_info: "Model",
+        model_file: PathType,
+        model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
+    ):
+        super().__init__(name, {}, {})
+        self._model_store = model_store
+        self._model_info = model_info
+        self._model_file = model_file
+        self._function_name = function_name
+
+    @property
+    def num_replica(self) -> int:
+        return max(int(self.resource_quota.cpu), 1)
+
+    @property
+    def required_models(self) -> t.List[Tag]:
+        return [self._model_info.tag]
+
+    # pylint: disable=attribute-defined-outside-init
+    def _setup(self) -> None:
+        with open(self._model_file, "rb") as f:
+            self._model = pickle.load(f)
+
+        if self._function_name == "__call__":
+            self._infer_func = self._model
+        else:
+            self._infer_func = getattr(self._model, self._function_name)
+
+    def _run(  # type: ignore[reportIncompatibleMethodOverride]
+        self,
+        inputs: t.Any,
+    ) -> t.Any:
+        return self._infer_func(inputs)
+
+
+@inject
+def load_runner(
+    tag: t.Union[str, Tag],
+    *,
+    name: t.Optional[str] = None,
+    model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
+) -> t.Union[_PicklableModelRunner, _PicklableModelSimpleRunner]:
+    """
```

Comment:
Could you also update the frameworks page in docs, so the inline docstring here show up in the frameworks documentation page: https://docs.bentoml.org/en/latest/frameworks/index.html